website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -110,13 +110,50 @@ As a tip, most command-line tools have a `--help` flag to show available command
 - `dbt run --help`: Lists the flags available for the `run` command
 :::
  
-### SQLFluff integration
-The dbt Cloud CLI supports [SQLFluff](https://sqlfluff.com/), a modular and configuration SQL linter, which warns you of complex functions, syntax, formatting, and compilation errors.
+### Lint SQL files 
 
-To get started, run `dbt sqlfluff -h` to see the list of supported commands and flags, such as `dbt sqlfluff lint` to lint SQL files.
+From the dbt Cloud CLI, you can invoke [SQLFluff](https://sqlfluff.com/) which is a modular and configurable SQL linter that warns you of complex functions, syntax, formatting, and compilation errors. The available SQLFluff commands are: `fix`, `format`, and `lint`. To show detailed information about these commands and the available flags, run the `dbt sqlfluff -h` command. 
+
+The `lint` command in dbt supports many of the same flags that you can pass to SQLFluff, except for those that might pose a security risk. To lint SQL files, run the command as follows:  
```

Comment:
Perhaps, "Running the `lint` command from the dbt Cloud CLI supports many of the same flags that you can pass to SQLFluff." to clarify dbt vs. the cloud cli.

Reply:
+1, might be worth adding that the `fix` and `format` commands, in addition to `lint`, support many of the same flags:
```
Running the supported SQLFluff commands from the dbt Cloud CLI supports many of the same flags that you can pass to SQLFluff, except those that might pose a security risk.

To lint SQL files, run the command as follows:
```